### PR TITLE
Modify infrasim version check for new QEMU version display.

### PR DESCRIPTION
**Pull request by JuneZhou19**
@InfraSIM/infrasim_dev

### CODE REVIEW

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_infrasim_ver 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
qemu version is 'QEMU      : QEMU emulator version infrasim-qemu_2.6.2-1.2.41ubuntu16.04.1', not expected

--------------------- >> end captured stdout << ---------------------- 
```

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_ipmi_console_stops_followed_by_node_destroy 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
[ 12022  ] test-socat starts to run
[ 12025  ] test-bmc starts to run
[ 12034  ] test-node is running
[ 12045  ] test-racadm starts to run
[ 12045  ] test-racadm stop
[ 12034  ] test-node stop
[ 12025  ] test-bmc stop
[ 12022  ] test-socat stop
Node test runtime workspace is destroyed.

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Ipmi-console: INFO: SIGTERM is sent to pid: 11813
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [ 11808  ] test-racadm stop
1Model: INFO: [ 11797  ] test-node stop
1Model: INFO: [ 11788  ] test-bmc stop
1Model: INFO: [ 11785  ] test-socat stop
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/disk00.img
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 12022  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 12025  ] test-bmc starts to run
1Model: INFO: [ 12034  ] test-node is running
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 12045  ] test-racadm starts to run
1Model: INFO: [ 12045  ] test-racadm stop
1Model: INFO: [ 12034  ] test-node stop
1Model: INFO: [ 12025  ] test-bmc stop
1Model: INFO: [ 12022  ] test-socat stop
1Model: INFO: [Node] Node test runtime workspcace is destroyed
--------------------- >> end captured logging << --------------------- 
TestName: test_infrasim_ver 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
qemu version is 'QEMU emulator version infrasim-qemu_2.6.2-1.2.41ubuntu16.04.1', not expected

--------------------- >> end captured stdout << ---------------------- 
```

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_ipmi_console_stops_followed_by_node_destroy 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
[ 21332  ] test-socat starts to run
[ 21335  ] test-bmc starts to run
[ 21344  ] test-node is running
[ 21355  ] test-racadm starts to run
[ 21355  ] test-racadm stop
[ 21344  ] test-node stop
[ 21335  ] test-bmc stop
[ 21332  ] test-socat stop
Node test runtime workspace is destroyed.

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
1Ipmi-console: INFO: SIGTERM is sent to pid: 21123
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [ 21118  ] test-racadm stop
1Model: INFO: [ 21106  ] test-node stop
1Model: INFO: [ 21097  ] test-bmc stop
1Model: INFO: [ 21094  ] test-socat stop
1Model: INFO: [compute] infrasim can't find numactl in this environment
1Model: INFO: [BaseDrive] Creating drive: /home/infrasim/.infrasim/test/disk00.img
1Model: INFO: test-socat's command line: /usr/bin/socat pty,link=/home/infrasim/.infrasim/test/.pty0,waitslave unix-listen:/home/infrasim/.infrasim/test/.serial,fork
1Model: INFO: [ 21332  ] test-socat starts to run
1Model: INFO: test-bmc's command line: ipmi_sim -c /home/infrasim/.infrasim/test/etc/vbmc.conf -f /home/infrasim/.infrasim/test/data/dell_r730.emu -n -s /home/infrasim/.infrasim/test/data
1Model: INFO: [ 21335  ] test-bmc starts to run
1Model: INFO: [ 21344  ] test-node is running
1Model: INFO: test-racadm's command line: racadmsim test 0.0.0.0 10022 admin admin auto
1Model: INFO: [ 21355  ] test-racadm starts to run
1Model: INFO: [ 21355  ] test-racadm stop
1Model: INFO: [ 21344  ] test-node stop
1Model: INFO: [ 21335  ] test-bmc stop
1Model: INFO: [ 21332  ] test-socat stop
1Model: INFO: [Node] Node test runtime workspcace is destroyed
--------------------- >> end captured logging << --------------------- 
TestName: test_infrasim_ver 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
qemu version is 'QEMU emulator version infrasim-qemu_2.6.2-1.2.41ubuntu16.04.1', not expected

--------------------- >> end captured stdout << ---------------------- 
```

**Comment by JuneZhou19** [comment]
retest please

File: test/functional/test_cli_commands.py [hunk 1]
```diff
@@ -271,6 +271,7 @@ def test_start_info_destroy_info(self):
         assert "Node {} runtime workspace doesn't exist".format(self.node_name) in output_info['destroy'][1]
 
     def test_init(self):
+        return
```
Comment (fub2) [inline test/functional/test_cli_commands.py]: Should not be included.

**Comment by JuneZhou19** [comment]
@fub2 correct, my bad. that was for debugging. Now removed.

**Comment by JuneZhou19** [comment]
retest this please

**Comment by emcinfrasim** [comment]
```
******* Functional Test ******* 
TestName: test_infrasim_ver 
ErrorType: exceptions.AssertionError 
ErrorMessage: 
-------------------- >> begin captured stdout << ---------------------
qemu version is 'QEMU emulator version infrasim-qemu_2.6.2-1.2.41ubuntu16.04.1, Copyright (c) 2003-2008 Fabrice Bellard', not expected

--------------------- >> end captured stdout << ---------------------- 
```

**Pull request closed without merging**